week-1/fix/median.js

```diff
@@ -3,9 +3,18 @@
 // If you're in the week-1 directory, you can run npm test -- fix to run the tests in the fix directory
 
 function calculateMedian(list) {
-  const middleIndex = Math.floor(list.length / 2);
-  const median = list.splice(middleIndex, 1)[0];
-  return median;
+    const middleIndex = (list.length + 1)/2; 
+
+  if(list.length%2 !== 0){
+    return list[middleIndex-1];
+  }
+  else{
+    const firstIndex = Math.floor(middleIndex);
+    const secondIndex = Math.round(middleIndex);
+     
+    return (list[firstIndex-1]+list[secondIndex-1])/2; 
+     
+  }
 }
 
```

Comment:
Great work Rabia, but I don't think you pushed the test file for the median exercise. Can you create a test file and create some test cases to validate that this function is working as it should?